client/src/theme/styles/components/global/checkboxes.js

```diff
@@ -54,6 +56,10 @@ export default `
         color: var(--color-base-neutral90);
       }
 
+      .checkbox--white & {
+        top: -1px;
+      }
+
```

Comment:
Not sure why, but the white ones look visually different than the gray ones after adding the border.